src/components/synchronization-page/DeletedObjectsPage.jsx

```diff
@@ -20,46 +20,49 @@ class DeletedObjectsPage extends React.Component {
         },
     };
 
-    model = {
-        getInitialSorting: () => ["deletedAt", "desc"],
-        getColumns: () => [
-            { name: "id", text: i18n.t("Identifier"), sortable: true },
-            { name: "code", text: i18n.t("Code"), sortable: true },
-            { name: "klass", text: i18n.t("Metadata type"), sortable: true },
-            { name: "deletedAt", text: i18n.t("Deleted date"), sortable: true },
-            { name: "deletedBy", text: i18n.t("Deleted by"), sortable: true },
-        ],
-        getDetails: () => [
-            { name: "id", text: i18n.t("Identifier") },
-            { name: "code", text: i18n.t("Code") },
-            { name: "klass", text: i18n.t("Metadata type") },
-            { name: "deletedAt", text: i18n.t("Deleted date") },
-            { name: "deletedBy", text: i18n.t("Deleted by") },
-        ],
-        getGroupFilterName: () => null,
-        getLevelFilterName: () => null,
-        getMetadataType: () => "deletedObjects",
-        getD2Model: () => ({
-            displayName: "Deleted Objects",
-            modelValidations: {
-                updatedAt: { type: "DATE" },
-            },
-        }),
-    };
+    models = [
+        {
+            getInitialSorting: () => ["deletedAt", "desc"],
+            getColumns: () => [
+                { name: "id", text: i18n.t("Identifier"), sortable: true },
+                { name: "code", text: i18n.t("Code"), sortable: true },
+                { name: "klass", text: i18n.t("Metadata type"), sortable: true },
+                { name: "deletedAt", text: i18n.t("Deleted date"), sortable: true },
+                { name: "deletedBy", text: i18n.t("Deleted by"), sortable: true },
+            ],
+            getDetails: () => [
+                { name: "id", text: i18n.t("Identifier") },
+                { name: "code", text: i18n.t("Code") },
+                { name: "klass", text: i18n.t("Metadata type") },
+                { name: "deletedAt", text: i18n.t("Deleted date") },
+                { name: "deletedBy", text: i18n.t("Deleted by") },
+            ],
+            getGroupFilterName: () => null,
+            getLevelFilterName: () => null,
+            getMetadataType: () => "deletedObjects",
+            getD2Model: () => ({
+                displayName: "Deleted Objects",
+                modelValidations: {
+                    updatedAt: { type: "DATE" },
+                },
+            }),
+        },
+    ];
 
     render() {
         const { d2 } = this.props;
 
         const title = i18n.t("Deleted Objects Synchronization");
+        const buttonLabel = <DeleteIcon />;
```

Comment:
AFIK, `DeleteIcon` is always the same, but `<DeleteIcon />` instances a new component each time.

Reply:
Oh! Since we ask for a component in ObjectsTable moving it outside the render method should fix it then.